test/e2e/rules_test.go

```diff
@@ -0,0 +1,123 @@
+// +build integration
+
+package e2e
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/efficientgo/e2e"
+	"github.com/efficientgo/tools/core/pkg/testutil"
+)
+
+func TestRulesAPI(t *testing.T) {
+	t.Parallel()
+
+	e, err := e2e.NewDockerEnvironment(envRulesAPIName)
+	testutil.Ok(t, err)
+	t.Cleanup(e.Close)
+
+	prepareConfigsAndCerts(t, rules, e)
+	_, token, rateLimiterAddr := startBaseServices(t, e, rules)
+	_, _, rulesEndpoint, _ := startServicesForMetrics(t, e) // TODO: create another function just for rules?
+
+	api, err := newObservatoriumAPIService(
+		e,
+		withMetricsEndpoints("", "", "http://"+rulesEndpoint),
```

Comment:
I think we might need to provide some write/read endpoint, otherwise API will complain that neither metrics nor logs endpoints are provided?

Reply:
I think for the test environment, since is using https://github.com/observatorium/api/blob/main/test/e2e/services.go#L224 this does not apply? It seems to append in case they are not empty: https://github.com/observatorium/api/blob/main/test/e2e/services.go#L250-L253 but no validation in case they are empty.

I also searched in the test output for this error message `"Neither logging nor metrics endpoints are enabled. Specifying at least a logging or a metrics endpoint is mandatory"` from https://github.com/observatorium/api/blob/main/main.go#L199-L202 but I haven't found any matches